go/interpreter/lfvm/instructions_test.go

```diff
@@ -2036,6 +1912,86 @@ func TestOpExtCodeHash_WritesHashOnStackIfAccountExists(t *testing.T) {
 	}
 }
 
+func TestInstructions_MCopy_ReturnsErrorOnFailure(t *testing.T) {
+
+	tests := map[string]struct {
+		destOffset, offset, size uint64
+		expectedError            error
+		gasRemoved               uint64
+	}{
+		"does nothing if size is zero": {
+			destOffset: 1024, offset: 125, size: 0,
```

Comment:
this test case is only checking that no error is produced, not that "does nothing". Perhaps this case should be its own test.

Reply:
as discussed, new test is written